Libs/MRML/Core/vtkMRMLScalarVolumeDisplayNode.cxx

```diff
@@ -796,7 +796,7 @@ void vtkMRMLScalarVolumeDisplayNode::CalculateAutoLevels()
 
     window = max-min;
     level = 0.5*(max+min);
-    lower = this->GetLevel();
+    lower = range[0];
```

Comment:
I don't think it's a huge issue either way, but maybe the best is to use the current value of Level.  That seems to fix the bug and be mostly consistent with the old behavior.